lib/migration-chunks/validation/field.js

```diff
@@ -10,6 +10,122 @@ const invalidActionError = (message, step) => {
   };
 };
 
+function fieldId (step) {
```

Comment:
My personal preference would be to enforce `func-style: expression` in the repo to keep things consistent across the board since we already enforce `no-use-before-define` and function declarations don't really serve any other purpose